eng/common/scripts/Prepare-Release.ps1

```diff
@@ -27,7 +26,7 @@ function Get-ReleaseDay($baseDate)
 
 $ErrorPreference = 'Stop'
 
-$packageProperties = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $serviceDirectory
```

Comment:
Lets always pass in the servicedirectory and if they don't set it that is fine we will look through everything but if they do pass it then the search will be a little more efficient.